waspc/data/Generator/templates/react-app/src/index.js

```diff
@@ -1,18 +1,18 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
-import { ReactQueryCacheProvider } from 'react-query'
+import { QueryClientProvider } from 'react-query'
 
 import router from './router'
-import queryCache from './queryCache'
+import { queryClient } from './queryClient'
 import * as serviceWorker from './serviceWorker'
 
 import './index.css'
 
 
 ReactDOM.render(
-  <ReactQueryCacheProvider queryCache={queryCache}>
+  <QueryClientProvider client={queryClient}>
```

Comment:
Reference: https://react-query.tanstack.com/guides/migrating-to-react-query-3#reactqueryconfigprovider-and-reactquerycacheprovider-have-both-been-replaced-by-queryclientprovider